src/components/app/data/hooks/useEnterpriseCourseEnrollments.js

```diff
@@ -73,7 +73,7 @@ export default function useEnterpriseCourseEnrollments(queryOptions = {}) {
     },
     enabled: isEnabled,
   });
-
+  // TODO: Talk about how we don't have access to weeksToComplete on the dashboard page.
```

Comment:
[curious] Should we backlog a ticket as a follow-up story to make the Dashboard start dates have parity to other start dates with regard to `weeksToComplete`? Maybe include the ticket number in the TODO comment?

Reply:
We can tackle this in the follow-up PR for coverage tomorrow morning.